pkg/process/net/common.go

```diff
@@ -378,6 +380,56 @@ func (r *RemoteSysProbeUtil) GetPprof(path string) ([]byte, error) {
 	return io.ReadAll(res.Body)
 }
 
+// GetServiceDiscoveryOpenPorts returns open ports from system-probe.
+func (r *RemoteSysProbeUtil) GetServiceDiscoveryOpenPorts(ctx context.Context) (*servicediscoverymodel.OpenPortsResponse, error) {
+	url := fmt.Sprintf("http://unix/%s/open_ports", sysconfig.ServiceDiscoveryModule)
```

Comment:
I'd suggest following the pattern of the other module URLs which are defined in `common_linux.go` and `common_windows.go` since the host part of the URL is platform-dependent.